--- liblastfm-0.3.0/configure	2009-06-04 16:41:27.000000000 +0000
+++ liblastfm-0.3.0/configure	2009-09-20 09:52:38.000000000 +0000
@@ -26,6 +26,18 @@
     puts "Will install to: "+ENV['LFM_PREFIX']
   end
 
+  h2 'Determining libdir' do
+    if ARGV.include? '--libdir'
+      n=ARGV.index '--libdir'
+      ENV['LFM_LIBDIR'] = ARGV[n+1]
+    end
+    ENV['LFM_LIBDIR'] = ENV['LFM_PREFIX']+'/lib' if ENV['LFM_LIBDIR'].nil?
+    if File.exists? ENV['LFM_LIBDIR'] and !File.directory? ENV['LFM_LIBDIR']
+      abort "Library directory exists but isn't a directory: "+ENV['LFM_LIBDIR']
+    end
+    puts "Library directory: "+ENV['LFM_LIBDIR']
+  end
+
   h1 'Generating Build System'
 
   h2 'Generating .qmake.env' do
--- liblastfm-0.3.0/admin/Makefile.rb	2009-06-04 16:41:27.000000000 +0000
+++ liblastfm-0.3.0/admin/Makefile.rb	2009-09-20 09:56:24.000000000 +0000
@@ -43,6 +43,8 @@
 $headers=''
 $install_prefix = ENV['LFM_PREFIX']
 abort("Environment variable LFM_PREFIX not defined") if $install_prefix.nil?
+$install_libdir = ENV['LFM_LIBDIR']
+abort("Environment variable LFM_LIBDIR not defined") if $install_libdir.nil?
 
 puts <<-EOS
 .PHONY: all
@@ -126,7 +128,7 @@
 
 .PHONY: install
 install: #{$install_headers} $(DESTDIR)#{$install_prefix}/include/lastfm.h
-	cd src && make install "INSTALL_ROOT=$(DESTDIR)#{$install_prefix}"
-	cd src/fingerprint && make install "INSTALL_ROOT=$(DESTDIR)#{$install_prefix}"
+	cd src && make install "INSTALL_ROOT=$(DESTDIR)#{$install_libdir}"
+	cd src/fingerprint && make install "INSTALL_ROOT=$(DESTDIR)#{$install_libdir}"
 
 EOS
\ Brak znaku nowej linii na końcu pliku
--- liblastfm-0.3.0/src/lastfm.pro	2009-06-04 16:41:27.000000000 +0000
+++ liblastfm-0.3.0/src/lastfm.pro	2009-09-20 10:15:31.000000000 +0000
@@ -18,4 +18,4 @@
     }
 }
 
-target.path = /lib
\ Brak znaku nowej linii na końcu pliku
+target.path = /
\ Brak znaku nowej linii na końcu pliku
--- liblastfm-0.3.0/src/fingerprint/fingerprint.pro	2009-06-04 16:41:27.000000000 +0000
+++ liblastfm-0.3.0/src/fingerprint/fingerprint.pro	2009-09-20 10:18:18.000000000 +0000
@@ -26,4 +26,4 @@
 }
 
 INSTALLS = target
-target.path = /lib
+target.path = /
